src/Swarm/TUI/View.hs

```diff
@@ -217,17 +217,24 @@ drawNewGameMenuUI (l :| ls) =
     let bestRealTime = if oneBest then "best:" else "best real time:"
     let noSame = if oneBest then const Nothing else Just
     let lastText = let la = "last:" in padRight (Pad $ T.length bestRealTime - T.length la) (txt la)
-    vBox . catMaybes $
-      [ Just $ txtWrap (nonBlank (s ^. scenarioDescription))
-      , Just $
-          padTop (Pad 3) $
-            padRight (Pad 1) (txt bestRealTime) <+> describeStatus (si ^. scenarioBestTime)
-      , noSame $ -- hide best game time if it is same as best real time
-          padTop (Pad 1) $
-            padRight (Pad 1) (txt "best game time:") <+> describeStatus (si ^. scenarioBestTicks)
-      , Just $
-          padTop (Pad 1) $
-            padRight (Pad 1) lastText <+> describeStatus (si ^. scenarioStatus)
+    vBox
+      [ txtWrap (nonBlank (s ^. scenarioDescription))
+      , vBox . catMaybes $
+          [ padTop (Pad 1)
```

Comment:
Why use the inner `vBox`?

If we wanted to get something out of it, we could at least refactor out the padding:
```Haskell
    let padTop1Right1 = padTop (Pad 1) . padRight (Pad 1)
    vBox
      [ txtWrap (nonBlank (s ^. scenarioDescription))
      , vBox . catMaybes . fmap padTop1Right1 $
```

Reply:
Hmm, I don't know, I'll put it back into a big `vBox`.

Factoring out `padTop (Pad 1) . padRight (Pad 1)` doesn't actually work, since the `padRight` calls are not being applied to the entire widget.